.vscode/extensions.json

```diff
@@ -1,3 +1,8 @@
 {
-  "recommendations": ["yzhang.markdown-all-in-one"]
+  "recommendations": [
+    "yzhang.markdown-all-in-one",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
```

Comment:
@chris-pearce wouldn't this only apply to JS based SDKs?

Reply:
Yeah, it's not needed in this repo. Will remove.

Reply:
@DaveOrDead https://github.com/kinde-oss/.github/pull/6/commits/8549b2e771494676a0f1acce2c746a9ba0b3b819. It also includes instructions on SDK repo naming and fixing the path to the Kinde logo after moving docs to the `.github` folder.